05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
-# Specifying sample designs in srvyr {#c05}
\ No newline at end of file
+# Specifying sample designs and replicate weights in srvyr {#c05}
+
+The primary reason for using packages like {survey} and {srvyr} are to incorporate the sampling design or replicate weights into estimates. By incorporating the sampling design or replicate weights, precision estimates (e.g., standard errors and confidence intervals) are appropriately calculated. In this chapter, we will introduce common sampling designs and common types of replicate weights, the mathematical methods for calculating estimates and standard errors for a given sampling design, and the R syntax to specify the sampling design or replicate weights. While we will show the math behind the estimates, the functions in these packages will do the calculation.
+
+The general process for estimation in the `srvyr` package is to:
+
+1. Create a `tbl_svy` object (a survey object) using: `as_survey_design` or `as_survey_rep`
+
+2. Subset data (if needed) using `filter` (subpopulations)
+
+3. Specify domains of analysis using `group_by` 
+
+4. Within `summarize`, specify variables to calculate including means, totals, proportions, quantiles, and more
+
+This chapter includes details on the first step - creating the survey object.
+
+## Common sampling designs
+
+A sampling design is the method used to draw a sample. Both logistical and statistical considerations are considered when developing a sampling design. When specifying a sampling design in R, the levels of sampling are specified along with the weights. Each record of a weight is constructed so that the particular record represents that many units in the population. For example, in a survey of 6th grade students in the United States, the weight associated with each responding student reflects how many students that record represents. Generally, the sum of the weights sum to the population total though some studies have the sum of the weights sum to the number of respondent records.
+
+Some common terminology across the designs are:
+
+- sample size, generally denoted as $n$, is the number of units selected
+- population size, generally denoted as $N$, is the number of units in the population
+- sampling frame is the list of units from which the sample is drawn
+
+### Simple random sample without replacement
+
+- Description: The simple random sample (SRS) without replacement is a sampling design where a fixed sample size is selected from a sampling frame, and every possible subsample has equal probability of selection.
+- Requirements: Must have a sampling frame for entire population
```

Comment:
Something to think about for all of these.  Some are complete sentences and have periods at the end (e.g., the description here and the disadvantages in the SRS w/replacement below).  Others don't have ending periods (e.g., the requirements line here).  I recommend being consistent on these, and it may be easiest just to add periods for all of them.  Just make sure they are all complete sentences.